### PR TITLE
Package opus.0.2.2

### PR DESCRIPTION
**Pull request by toots**
### `opus.0.2.2`
Bindings to libopus

---
* Homepage: https://github.com/savonet/ocaml-opus
* Source repo: git+https://github.com/savonet/ocaml-opus.git
* Bug tracker: https://github.com/savonet/ocaml-opus/issues

---
:camel: Pull-request generated by opam-publish v2.1.0

### CODE REVIEW

File: packages/opus/opus.0.2.2/opam [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  ["dune" "subst"] {dev}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
```
[the remark this answers is not in the document]
Comment (mseri) [reply packages/opus/opus.0.2.2/opam:28]: Actually it is a bit more complicated, @kit-ty-kate is sending a PR for this to your repo

**Comment by toots** [comment]
Thanks y'all. I have merged the PR and updated the release to incorporate it.

**Comment by toots** [comment]
Hey! Any update on this one? It fixes a memory leak with some ogg headers..

**Comment by kit-ty-kate** [comment]
The compilation seems to be failing on recent versions of gcc (on archlinux):
```
- ogg_stubs.c: In function ‘ocaml_ogg_sync_pageseek’:
- ogg_stubs.c:170:26: error: expected declaration specifiers or ‘...’ before string constant
-   170 |   CAMLparam2(callback, oy);
```
Could you have a look?

**Comment by toots** [comment]
> The compilation seems to be failing on recent versions of gcc (on archlinux):
> 
> ```
> - ogg_stubs.c: In function ‘ocaml_ogg_sync_pageseek’:
> - ogg_stubs.c:170:26: error: expected declaration specifiers or ‘...’ before string constant
> -   170 |   CAMLparam2(callback, oy);
> ```
> 
> Could you have a look?

I definitely will but this seems related to `ogg` and out of scope in the context of updating the `opus` package..

**Comment by kit-ty-kate** [comment]
Ah fair enough, I didn’t notice it was the other package (the names are close to be fair ^^")

Sorry for the wait. Thanks

**Comment by toots** [comment]
No pb thanks. The error looks more like an issue with the `archlinux` setup, the code failing is pretty standard call to `CAMLparam2` and fails in both `opus` and `ogg`..

**Pull request merged**